envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/RoutesAssertions.kt

```diff
@@ -45,13 +45,13 @@ fun RouteAction.hasCustomRequestTimeout(requestTimeout: Duration): RouteAction {
     return this
 }
 
-fun RouteAction.autoHostRewriteHeaderIsEmpty(): RouteAction {
-    assertThat(this.autoHostRewriteHeader).isEmpty()
+fun RouteAction.hostRewriteHeaderIsEmpty(): RouteAction {
```

Comment:
this was renamed with backwards compatiblity: https://github.com/envoyproxy/envoy/blob/d321ede4d262cc2259064337be7b407e463ede24/generated_api_shadow/envoy/api/v2/route/route_components.proto#L869-L872